linux_os/guide/system/logging/journald/socket_systemd-journal-remote_disabled/rule.yml

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
+
+title: 'Disable systemd-journal-remote Socket'
+
+description: |-
+    Journald supports the ability to receive messages from remote hosts,
+    thus acting as a log server. Clients should not receive data from
+    other hosts.
+    NOTE:
+        The same package, systemd-journal-remote , is used for both sending
+        logs to remote hosts and receiving incoming logs.
+        With regards to receiving logs, there are two Systemd unit files;
+        systemd-journal-remote.socket and systemd-journal-remote.service.
+
+rationale: |-
+    If a client is configured to also receive data, thus turning it into
+    a server, the client system is acting outside it's operational boundary.
+
+severity: medium
+
+references:
+    cis@ubuntu2204: 4.2.1.1.4
+
+ocil_clause: 'the systemd-journal-remote socket is running'
+
+ocil: |-
+    {{{ ocil_service_enabled(service="systemd-journal-remote.socket") }}}
+
+fixtext: |-
+    {{{ fixtext_service_disabled("systemd-journal-remote.socket") }}}
```

Comment:
Here goes the same as for the ocil above. I think you need a new macro or input the fixtext manually so that the terminology is not mixed up.

Reply:
created a new macro for it